net/BigBuffers.Tests.Monster/BigBuffers.Tests.Monster.csproj

```
@@ -54,33 +54,33 @@
       <IsFirstBuild Condition="$(IsFirstBuild)">false</IsFirstBuild>
     </PropertyGroup>
   </Target>
-  
+
   <Target Name="GenerateModelsFromSchema" DependsOnTargets="DetectFirstBuild" Condition="$(IsFirstBuild)" BeforeTargets="DispatchToInnerBuilds;PrepareForBuild;BeforeCompile;Compile;CoreCompile;BeforeBuild;Build;BeforeRebuild;Rebuild">
     <Message Importance="normal" Text="Generating models from schema..." />
-    
+
     <!-- make the generated dir if it doesn't exist -->
     <MakeDir Directories="Generated" />
 
     <!-- delete any existing generated models -->
     <ItemGroup>
       <FilesToDelete Include="Generated/**/*.cs" />
     </ItemGroup>
-    
+
     <!--<Message Text="Deleting: @(FilesToDelete, ', ')" Importance="high" />-->
-    
+
     <Delete Files="@(FilesToDelete)" />
-    
+
     <!-- locate bufc and schema files -->
     <ItemGroup>
       <LocOfBufC Include="../../cmake-build-$(Configuration.ToLower())/bufc" />
-      <LocOfBufC Include="../../cmake-build-$(Configuration.ToLower())/bufc.exe" />
+      <LocOfBufC Include="../../cmake-build-$(Configuration.ToLower())/$(Configuration.ToLower())/bufc.exe" />
```

```suggestion
      <LocOfBufC Include="../../cmake-build-$(Configuration.ToLower())/bufc.exe" />
      <LocOfBufC Include="../../cmake-build-$(Configuration.ToLower())/$(Configuration.ToLower())/bufc.exe" />
```

This should be adding to the search locations instead of changing the existing ones.